quickstarts/rest/Search_Grounding.ipynb

```diff
@@ -41,7 +41,7 @@
         "\n",
         "<table align=\"left\">\n",
         "  <td>\n",
-        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/google-gemini/cookbook/blob/main/quickstarts/rest/Search_Grounding.ipynb\"><img src=\"../images/colab_logo_32px.png?raw=1\" />Run in Google Colab</a>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/google-gemini/cookbook/blob/main/quickstarts/rest/Search_Grounding.ipynb\"><img src=\"https://github.com/google-gemini/cookbook/blob/main/quickstarts/images/colab_logo_32px.png?raw=1\" />Run in Google Colab</a>\n",
```

Comment:
This image 404s. Colab sometimes "fixes" your links for you - might need to revert it.